docs/src/pages/docs/routing.mdx

```diff
@@ -46,6 +46,23 @@ Still, in case you're defining other routing config, make sure to keep them in s
 
 </Details>
 
+<Details id="typescript-version-error">
+  <summary>What if I encounter a TypeScript error like `TS2554: Expected 0 arguments, but got 1`?</summary>
+
+  If you're seeing TypeScript errors such as `TS2554: Expected 0 arguments, but got 1` when using `defineRouting` or other routing functions, this may be due to an incompatible TypeScript version. To fix this, ensure that your project is using **TypeScript version 5 or later**.
+
+  You can upgrade your TypeScript version by running:
+
+  ```sh
+  npm install typescript@latest
+  # or with yarn
+  yarn add typescript@latest
+  ```
+
+  After upgrading, restart your editor or TypeScript server to ensure the latest types are loaded.
+
+</Details>
```

Comment:
I think due to `next-intl@4` being on the horizon with the peer dependency check for TypeScript 5, I think we might be able to leave this section out for now, as it shouldn't be required anymore once that version is out. 

As a side note, I just checked and TypeScript 5 was released on March 16, 2023. I'd hope that most people who use App Router features from `next-intl` use somewhat up-to-date dependencies, as Next.js really pushes for that.

From my experience, developers often also search in issues & discussions for terms like "TS2554: Expected 0 arguments, but got 1", so a mention of this error in GitHub comments often also goes a long way without having to include it in the docs (see [docs guidelines](https://github.com/amannn/next-intl/blob/main/CONTRIBUTORS.md#documentation-guidelines)).

Does that sound reasonable to you?

Reply:
Thank you very much for your explanations 🙏
I agree with everything you say, it seems reasonable to me.

Regarding the search for issues and discussions about the error, for my part I haven't found anything on GitHub (or even on the internet in general, apart from the StackOverflow discussion mentioned in my PR) 🤔

I admit I'm not very familiar with the `Discussions` part of GitHub. Are you suggesting that I open a discussion to share the solution with the community? 👀 

Anyway, no problem not including that in the documentation! ✨

Reply:
> PS: This part of the documentation has been removed via [018e3a2](https://github.com/amannn/next-intl/pull/1538/commits/018e3a2e4a6a3a63db9c029f895ed90ea92f1e53)



Great, thanks! 

> I finally found [this discussion](https://github.com/amannn/next-intl/discussions/1436) 🎉

Ah yep, that seems to be the same one! I've marked the latest comment as the answer there now, hopefully that will help in case other people run into this.